pollux/sql-doobie/src/main/scala/org/hyperledger/identus/pollux/sql/repository/JdbcCredentialStatusListRepository.scala

```diff
@@ -212,9 +212,24 @@ class JdbcCredentialStatusListRepository(xa: Transactor[ContextAwareTask], xb: T
     } yield ()
   }
 
-  def getCredentialStatusListsWithCreds: UIO[List[CredentialStatusListWithCreds]] = {
+  def getCredentialStatusListIds: UIO[Seq[(WalletId, UUID)]] = {
+    val cxnIO =
+      sql"""
+           | SELECT 
+           |    wallet_id,
+           |    id
+           |  FROM public.credential_status_lists
+           |""".stripMargin
+        .query[(WalletId, UUID)]
+        .to[Seq]
```

Comment:
This code also might be a potential bottleneck when the number of records in the table grows.

Reply:
Yep, already identified as part of the recent Scalability investigation work.